src/elements/hx-checkbox-set/index.spec.js

```diff
@@ -0,0 +1,101 @@
+import { fixture, oneEvent, expect } from '@open-wc/testing';
+
+/**
+ * <hx-checkbox-set> component tests
+ *
+ * @type HXCheckboxSetElement
+ *
+ */
+describe('<hx-checkbox-set> component tests', () => {
+    const template = '<hx-checkbox-set>';
+
+    describe('instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const component = /** @type {HXCheckboxSetElement} */ await fixture(template);
+            const attr = component.hasAttribute('hx-defined');
+
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const component = /** @type {HXCheckboxSetElement} */ await fixture(template);
+            const prop = component.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered light DOM should NOT equal simple template ${template}`, async () => {
+            const component = /** @type {HXCheckboxSetElement} */ await fixture(template);
+
+            expect(component).lightDom.to.not.equal(template);
+        });
+
+        it(`should NOT have a Shadow DOM`, async () => {
+            const component = /** @type {HXCheckboxSetElement} */ await fixture(template);
+            const shadow = component.shadowRoot;
+
+            expect(shadow).to.be.null;
+        });
+    });
+
+    describe('test <hx-checkbox-set> getters and setters', () => {
+        it('should be able to set isDirty', async () => {
+            const component = /** @type {HXCheckboxSetElement} */ await fixture(template);
+            const attr = component.isDirty;
+
+            expect(attr).to.be.false;
+        });
+
+        it('should be able to set wasChanged', async () => {
+            const component = /** @type {HXCheckboxSetElement} */ await fixture(template);
+            const attr = component.wasChanged;
+
+            expect(attr).to.be.false;
+        });
+
+        it('should be able to set wasTouched', async () => {
```

Comment:
Please change description, since we want to get the state rather than set it: 

```js
it('should be able to get component wasTouched state', async () => {
```

Reply:
Changes have been done, Please review